articles/configure-your-markdown-engine.mdx

```diff
@@ -16,119 +16,164 @@ tags: []
 related_articles: []
 related_links:
 ---
-To store and edit Markdown content in CloudCannon, we need to process it into HTML to make it editable. You can use any Markdown engine you want when building your site, this only covers the Markdown content you edit inside CloudCannon.
+CloudCannon needs to parse Markdown content into HTML in order to preview and edit it. It's important for CloudCannon to save Markdown in the same format expected by your SSG, so that your content is built correctly.
```

Comment:
```suggestion
CloudCannon needs to parse Markdown content into HTML to preview and edit it. Your Markdown must be in a format recognized by your SSG so that CloudCannon can build your content.
```